Document RGB-NIR preprocessing and CLI improvements (#7)
scsd-cdh/payload-ai-suite: README.md

- Cross-reference FIRMS fire event/EoNet and query via Copernicus's process API.
- Run VGG model on labeled (fire/no fire) data.
- Preproces RGB-NIR algorithm

> - Optional NIR channel support for RGB-NIR fusion model.

Reviewer: ```suggestion
- Optional NIR channel support for RGB-NIR 4 channel tensor model.
```
Technically, the whole point of this feature is to circumvent the preprocess RGB-NIR fusion. We keep the 4th channel, not fuse it.

It has been shown that NIR wavelengths between 830 nm and 1000 nm, captured by COTS camera sensors, provide statistically significant advantages in fire detection. As commonly employed in the field of robotics, our thesis is that the accuracy of our model will increase with an RGB-NIR fusion image as an input to improve feature detection.

> If the `--nir-model` flag is used, preprocessing includes an additional NIR channel. Currently, this NIR data is simulated using one of the existing RGB channels as a proxy for NIR. This placeholder can later be replaced with true multispectral NIR data from the camera. 

Reviewer: ```suggestion
If the `--nir-model` flag is used, preprocessing will maintain the additional NIR channel for R&D purposes. Currently, this NIR data can be found in the alpha channel of the test data. In production, the input would be the raw bayer output of the multispectral camera. 
```
The proxy part is not quite how it works. When we are quering the copernicus api, we are getting the NIR band which is stored in the alpha channel. If we used a different file format like exr or envi we could actually just store the bands as is.

   ```bash
   python3 main.py --batch-download
   ```

> 5. **Run the wildfire classification model with simulated NIR**
>    '''bash
>    python3 main.py --run-model --nir-model
>    '''

Reviewer: ```suggestion
5. **Run the wildfire classification model with simulated NIR**
   ```bash
   python3 main.py --run-model --nir-model
   ```
```
To render code snippet you use back ticks not single quotes.
